setup.py

```diff
@@ -50,9 +50,12 @@ def exit_with_error(message):
         long_description = f.read()
 
     # Blosc version
-    VERSION = open('VERSION').read().strip()
+    with open('VERSION') as f:
+        VERSION = .read().strip()
```

Comment:
```suggestion
        VERSION = f.read().strip()
```
You didn't see this...